examples/simple_adding.rb

```diff
@@ -0,0 +1,10 @@
+def simple_adding(number)
+  sum = 0
+  number.times do |i|
+    i = i + 1
```

Comment:
the `times` method automatically increments your local variable named `i` as the loop iterates. no need to increment the variable yourself.